components/dashboard/script.js

```diff
@@ -0,0 +1,26 @@
+let fs = require('fs');
+
+let vars = {
+    REACT_APP_HOSTNAME : "Gitpod",
+    REACT_APP_HOST_URL : "http://developarm.devspaces.com/",
```

Comment:
This can't be hardcoded. this should come from ENV or config

Reply:
See The way I am doing it is you just need to change your desired values here in this file. So these are default values . This script writes a .env file. And then those are used elsewhere in the code. So in this this script.js is one point config spot.